sopel/tools/identifiers.py

```diff
@@ -137,10 +136,13 @@ def __init__(
         identifier: str,
         *,
         casemapping: Casemapping = rfc1459_lower,
+        chantypes: tuple = DEFAULT_CHANTYPES,
     ) -> None:
         super().__init__()
         self.casemapping: Casemapping = casemapping
         """Casemapping function to lower the identifier."""
+        self.chantypes = chantypes
+        """Tuple of prefix used for channels."""
```

Comment:
```suggestion
        """Tuple of prefixes used for channels."""
```

What do you call a nemesi**S** that also contains `e`? 🤔